modules/graphstudio/pages/design-schema.adoc

```diff
@@ -85,64 +86,66 @@ when accessing these attributes.
-vertex type are selected based on your clicking action. However, you can
-change that by choosing another vertex type in the dropdown list.
-
-You can also click the green plus sign at the right of the source and
-target vertex types section to add more source and target vertex types
-of the edge type.
-
-image:edge-multi-pair.png[image]
-
-By default, the edge type is undirected. To make the edge type directed,
-mark the Directed checkbox:
+The edge type name cannot be left blank.
 
-image:directed-edge.png[image]
-
-If Directed is checked, another checkbox will appear for you to choose
-whether the edge type should include reverse edges.
+By default, the edge type is undirected.
+To make the edge type directed, mark the Directed checkbox, which will bring up another checkbox will appear for you to choose whether the edge type should include reverse edges.
 Including reverse edges provides more flexibility when designing queries.
 
-Unselect the reverse edge checkbox ONLY IF your machine memory is very tight, because if there is no reverse edge, queries will not be able to traverse
+The source and target vertex types are selected based on the vertices you clicked, but you can change that by choosing another vertex type in the dropdown list.
+Click the green plus sign at the right of the source and
+target vertex types section to add more source and target vertex types.
+
+Unselect the reverse edge checkbox ONLY IF your machine memory is very limited, because if there is no reverse edge, queries will not be able to traverse
 backwards along this directed edge type from the target vertex to the
 source vertex.
 
-Editing edge type attributes is the same as editing vertex type
-attributes.
-
 Once you are satisfied with the edge type settings, click the checkmark button at the top to add the edge
 type, which will instantly appear in the working panel.
 
-You can add multiple edge types between the same source vertex type and
-target vertex type pair. Moreover, an edge can use the same vertex type
-for both its source vertex type and its target vertex type, e.g., a
-Friendship edge from a Person vertex to Person vertex.
+=== Edge attributes
+
+image::edge-attributes.png[]
+
+Add an attribute to your edge in the Attributes panel.
+Edge attributes work just like vertex attributes.
+
+The btn:[Discriminator] checkbox allows you to enable multiple edges with the same attribute type to exist between the same vertex pair.
+
+You cannot change an edge to become a Discriminator type if it is used in another graph.
+
+Non-Discriminator attributes can be added and dropped at will.
+However, modifying Discriminator attributes performs the same actions in the backend as dropping and recreating an edge.
+Therefore, data loaded to edges with Discriminator attributes will be lost upon attribute modification.
+
+[NOTE]
+Attribute values must be unique if multiple attributes of the same type exist between the same vertex pair.
```

Comment:
Alex I think that what you mean here is multiple edges of the same type exist between the same vertex pair? Attributes exist within vertices/edges not between vertices.
For single-edge(no discriminator attributes): only one edge of the edge type between any two vertices.
For multi-edge (with discriminator attributes): can have multiple edges of the edge type between any two vertices provided that the discriminators are unique.

Reply:
Yes, I meant edges, good catch!